### PR TITLE
DYN-2560 - Increase the code coverage: DynamoCore/Models Folder First Part

### PR DESCRIPTION
**Pull request by RobertGlobant20**
### Purpose

Increase the code coverage adding test cases for the DynamoCore/Models folder first part.
I created several unit tests using nunit and AxoCover (code coverage) for the next classes:

ZoomEventArgs
TaskDialogEventArgs
EvaluationCompletedEventArgs
DynamoModelUpdateArgs
FunctionNamePromptEventArgs
PresetsNamePromptEventArgs
ViewOperationEventArgs
PointEventArgs
WorkspaceEventArgs
ModelEventArgs
DynamoModelEvents

### Declarations

Check these if you believe they are true

- [X] The codebase is in a better state after this PR
- [X] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [X] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

@QilongTang 

### FYIs

@alfredo-pozo

### CODE REVIEW

**Comment by RobertGlobant20** [comment]
Screenshot about the Code Coverage generated by AxoCover just for classes in the files:
Models/DynamoModelEventArgs.cs
Models/DynamoModelEvents.cs

image

**Comment by RobertGlobant20** [comment]
I executed the all the test cases in the DynamoCoreTest project and I got the same results as in in previous releases, also all my new test cases were executed successfully.

image

File: test/DynamoCoreTests/Models/DynamoModelEventArgsTest.cs [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+            Assert.AreEqual(zoom2.Point.X, 200);
+            Assert.AreEqual(zoom2.Point.Y, 100);
+
+            //This will execute the bool hasPoint() method, it returns true because we passed the Point as parameter
+            Assert.IsTrue(zoom.hasPoint());
+        }
+
+        [Test]
+        [Category("UnitTests")]
```
Comment (QilongTang) [inline test/DynamoCoreTests/Models/DynamoModelEventArgsTest.cs:67]: Would you add some comments on this test?

Comment (RobertGlobant20) [reply test/DynamoCoreTests/Models/DynamoModelEventArgsTest.cs:73]: @QilongTang I added comments for this test case in the https://github.com/DynamoDS/Dynamo/pull/10612/commits/fd579961c315ade97b04f3634e7054724dc1ea9d commit.

Thanks

File: test/DynamoCoreTests/Models/DynamoModelEventsTest.cs [hunk 2]
```diff
@@ -0,0 +1,509 @@
...
+        [Test]
+        [Category("UnitTests")]
+        public void TestOnWorkspaceClearing()
+        {
+            //Arrange
+            //This will subscribe our local method to the RequestLayoutUpdate event
+            CurrentDynamoModel.WorkspaceClearing += CurrentDynamoModel_WorkspaceClearing;
+
+            //Act
+            //Internall this will execute the OnWorkspaceClearing() method in DynamoModelEvents
```
Comment (QilongTang) [inline test/DynamoCoreTests/Models/DynamoModelEventsTest.cs:141]: Spelling

**Comment by QilongTang** [comment]
LGTM with comments

File: test/DynamoCoreTests/Models/DynamoModelEventsTest.cs (hunk 2 above)
Comment (RobertGlobant20) [reply test/DynamoCoreTests/Models/DynamoModelEventsTest.cs]: @QilongTang 
I fixed the spelling in two test methods in the next commit:
https://github.com/DynamoDS/Dynamo/pull/10612/commits/22270346ba76b9fcdb4cd08be0be75ce65a1f326

**Pull request merged**